aip/general/0162.md

```diff
@@ -342,67 +252,39 @@ ranges of time.
 Revisions are sometimes expensive to store, and there are valid use cases to
 want to remove one or more revisions from a resource's revision history.
 
-APIs **may** define a method to delete revisions, with a structure similar to
-`Delete` ([AIP-135][]) methods:
+APIs **may** define a standard `Delete` ([AIP-135][]) method to delete resource revisions
 
-```proto
-rpc DeleteBookRevision(DeleteBookRevisionRequest)
-    returns (Book) {
-  option (google.api.http) = {
-    delete: "/v1/{name=publishers/*/books/*}:deleteRevision"
-  };
-}
-```
+- Resource revisions **should** be automatically deleted on parent resource deletion.
+- If resource revisions meant to have longer lifespan than the parent resource, resource and resource revision
+should be side by side resource collections.
```

Comment:
I don't know what this means. More detail definitely required.

Reply:
Move it to the top of the page where "abstract revisions as nested collections are not proper"